stable/kommander/values.yaml

```
@@ -151,12 +151,12 @@ grafana:
       datasources:
         - name: ThanosQuery
           type: prometheus
-          url: http://kommander-kubeaddons-thanos-query-http.kommander:10902/
+          url: http://kommander-kubeaddons-thanos-query-http.kommander.svc.cluster.local:10902/
```

As discussed, we don't need `.cluster.local` suffix here as `.svc` is included in `noproxy` documentation. Please remove it as the cluster domain is actually configurable in a kubernetes cluster (although this is rarely changed...).
```suggestion
          url: http://kommander-kubeaddons-thanos-query-http.kommander.svc:10902/
```

---

I have made the requested changes, please have another look.

The `cluster.local` suffix is used all across the repo, including this file. I have not changed this though as I think this is outside of the scope of this change.